app/dependencies.py

```diff
@@ -11,19 +11,22 @@
 GOOGLE_ERROR = config.CLIENT_SECRET_FILE is None
 APP_PATH = os.path.dirname(os.path.realpath(__file__))
 MEDIA_PATH = os.path.join(APP_PATH, config.MEDIA_DIRECTORY)
+EVENT_IMAGES_PATH = os.path.join(APP_PATH, config.EVENT_IMAGE_DIRECTORY)
```

Comment:
Prefer having `EVENT_IMAGE_DIRECTORY` as an absolute location.

If the user want to create a safe environment, he or she might choose to store the images in a location that has nothing to do with the directory of the application itself.

Also, prefer calling it `UPLOAD_PATH`.

Reply:
changed to UPLOAD_PATH.
I added a comment about specifiying a specific local path in the config. However for testing purposes I mentioned that they can specify a folder name and that will be created under app/ (I check whether they put a valid path and otherwise try to create a dir under the working directory).
I think this is easier at least for testing purposes but let me know what you think.